modules/installation-nutanix-config-yaml.adoc

```diff
@@ -94,8 +94,8 @@ endif::openshift-origin[]
 If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance.
 ====
 <4> Optional: Provide additional configuration for the machine pool parameters for the compute and control plane machines.
-ifndef::openshift-origin[]
 <5> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
+ifndef::openshift-origin[]
```

Comment:
@mjpytlak @mburke5678, any objections here?

Reply:
I encountered the same problem when picking up my Nutanix work this morning after a rebase. Pushing the `ifndef` to line 98 should resolve the error.

Reply:
@kalexand-rh My only concern is with the default value. For OKD it is OVNKubernetes; for OCP is it OpenShiftSDN, I do believe.

Reply:
Not sure @mburke5678 looks like the change was made wholesale by https://github.com/openshift/openshift-docs/pull/54319

Reply:
@mburke5678, OVNKubernetes appears to be the default as of 4.12, so it should be correct for both OKD and OCP.